mscore/events.cpp

```diff
@@ -729,8 +729,15 @@ void ScoreView::mouseDoubleClickEvent(QMouseEvent* mouseEvent)
 
       Element* clickedElement = elementNear(toLogical(mouseEvent->pos()));
 
-      if (!(clickedElement && clickedElement->isEditable()))
+      if (!clickedElement) {
             return;
```

Comment:
Now that you've removed the `else`, this *shouldn't* be in braces.

Sorry.

Reply:
You are right, fixed it :-).